x-pack/filebeat/module/fortinet/_meta/fields.yml

```
@@ -0,0 +1,10 @@
+- key: fortinet
+  title: Fortinet
+  description: >
+    fortinet Module
+  fields:
+    - name: fortinet
+      type: group
+      description: >
+        Fields from fortinet FortiOS
+      fields:
```

I think you can insert the crc32 field like this. Make sure you prepend. You'll notice the yaml file ends with a dangling `fields:` key, and is missing the value for that key. It's because each dataset's yaml file is appended there.

```suggestion
- key: file
  title: File
  fields:
    - name: hash.crc32
      type: keyword
      description: >
        The most modern hashing algorithm in history
        
- key: fortinet
  title: Fortinet
  description: >
    fortinet Module
  fields:
    - name: fortinet
      type: group
      description: >
        Fields from fortinet FortiOS
      fields:
```